app/vector_icons/convert_svg_to_icon.py

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import optparse
+import subprocess
+import os
+
+
+def process(input_file, output_file):
+    subprocess.run([
+        'npm', 'exec', '--prefix', '../../brave', 'skiafy', input_file,
+        output_file
+    ])
+
+
+def main():
+    parser = optparse.OptionParser()
+    (options, args) = parser.parse_args()
+
+    process(args[0], args[1])
```

Comment:
totally optional:

I'd prefer add_option than args, so that we can avoid making mistakes and give  more comprehensive -h output.